src/common/PeopleSection/index.js

```diff
@@ -0,0 +1,55 @@
+import { exampleConfiguration } from "./exampleConfiguration";
+import { examplePopularPeople } from "./examplePopularPeople";
+import { exampleCredits } from "./exampleCredits";
+import pictureSubstitution from "./pictureSubstitution.svg";
+import { PeopleList, PersonTile, ProfilePicture, Role, TileTitle, Title } from "./styled";
+
+const PeopleSection = ({ title }) => {
+    let peopleList;
+    const sectionName = title.toLowerCase();
+
+    switch (sectionName) {
+        case "cast":
+            peopleList = exampleCredits.cast;
+            break;
+
+        case "crew":
+            peopleList = exampleCredits.crew;
+            break;
+
+        default:
+            peopleList = examplePopularPeople.results;
+    }
```

Comment:
missing ;

Reply:
I think we are not supposed to put semicolons after switch if/else loops etc.  Or does eslint show it as an error? Still can't make it work on my laptop :/

Reply:
eslint shows its ok